fairlearn/metrics/_skm_wrappers.py

```diff
@@ -135,3 +135,40 @@ def internal_mse_wrapper(y_true, y_pred, sample_weight=None):
 
     return metric_by_group(internal_mse_wrapper,
                            y_true, y_pred, group_membership, sample_weight=sample_weight)
+
+def group_root_mean_squared_error(y_true, y_pred, group_membership, *,
```

Comment:
These should be going in to `test_group_sklearn_wrappers.py`

At the very list, they should be added to the appropriate "supported_metrics" things-which-aren't-dictionaries-like-the-comment-says at the top. Since they also take that `multioutput` argument, could you also add a test setting a different value of that, so that we have a test confirming that the argument is passed through OK? e.g. `test_group_mean_squared_error_multioutput()`

Reply:
Thank you for showing me the tests. Done.